numbert/reranker.py

```diff
@@ -0,0 +1,244 @@
+'''
+Module for providing python interface to numBERT re-rankers
+'''
+
+import os
+os.environ["JAVA_HOME"] = "/usr/lib/jvm/java-11-openjdk-amd64"
```

Comment:
Should we make the user to set its own JAVA_HOME? I mean, will this hard code path work in all cases?

Reply:
True, this might not necessarily generalize, I can just remove this and it won't affect anything, even my scripts.

Reply:
Resolved in c377d45